src/Sentry/Laravel/EventHandler.php

```diff
@@ -171,6 +194,23 @@ protected function queryExecutedHandler(QueryExecuted $query)
         ));
     }
 
+    /**
+     * Until Laravel 5.3
+     *
+     * @param string     $level
+     * @param string     $message
+     * @param array|null $context
+     */
+    protected function logHandler($level, $message, $context)
+    {
+        $this->client->breadcrumbs->record(array(
```

Comment:
Sorry, misclicked the approve.
We do not have `$this->client` anymore instead you need to call 
```
Integration::addBreadcrumb(
```